model/src/main/proto/spine_examples/shareaware/wallet/wallet.proto

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+syntax = "proto3";
+
+package spine_examples.shareaware.wallet;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.shareaware.spine.io";
+option java_package = "io.spine.examples.shareaware.wallet";
+option java_outer_classname = "WalletProto";
+option java_multiple_files = true;
+
+import "spine/core/user_id.proto";
+import "spine/money/money.proto";
+import "spine_examples/shareaware/identifiers.proto";
+
+// Represents an amount of user's money.
+message Wallet {
+    option (entity) = {kind: AGGREGATE};
+
+    // The ID of the wallet.
+    WalletId id = 1;
+
+    // The balance of the wallet.
+    spine.money.Money balance = 2;
+
+    // An amount of reserved money.
+    //
+    // This field is used for money safety while the withdrawal transaction is in process.
```

Comment:
This field should have a domain-specific explanation. E.g.:

"This amount is considered reserved, because there may be pending operations against this wallet,
that may imply spendings. It is not possible to use the reserved amount directly, other than in scope 
of these pending operations."

(BTW, I think this should eventually become a `map` between IDs of "operations" and reserved amounts. Let's touch this idea next time we speak, and in future PRs.)

In turn, `balance` field should have its own counterpart, explaining its usage. I encourage you to write the second paragraph for `balance` field by yourself.